sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -534,16 +537,18 @@ class EthService(
 
   @RpcMethod("debug_traceBlockByNumber")
   @RpcOptionalParameters(1)
-  def traceBlockByNumber(number: String, config: TraceOptions): DebugTraceBlockView = {
+  def traceBlockByNumber(number: String, config: TraceOptions): Any = {
     val hash: Hash = {
       applyOnAccountView { nodeView =>
-        Hash.fromBytes(idToBytes(getBlockIdByTag(nodeView, number)))}}
+        Hash.fromBytes(idToBytes(getBlockIdByTag(nodeView, number)))
+      }
+    }
     traceBlockByHash(hash, config)
   }
 
   @RpcMethod("debug_traceBlockByHash")
   @RpcOptionalParameters(1)
-  def traceBlockByHash(hash: Hash, config: TraceOptions): DebugTraceBlockView = {
+  def traceBlockByHash(hash: Hash, config: TraceOptions): Any = {
```

Comment:
why result type is `Any`?
I think we can be more accurate here, using `List[JsonNode]`

Reply:
Agree, return datatype changed to `List[JsonNode]`